### ISSUE TITLE
Is it possible to use FNC_SetRadio on the ACRE_PRC77?

### ISSUE THREAD

**Issue by Drofseh** [dklollol/Olsen-Framework-Arma-3#109]
The ACRE_PRC77 doesn't have channels and uses dials to adjust the frequency.
http://www.unitedoperations.net/wiki/AN/PRC-77#Operation

Is it possible to use FNC_SetRadio or some other function to set the band and frequency at mission start?

Thanks you.

**Comment by BlackHawkPL** [comment]
Sorry for late reply. I tried using function from ACRE's API and it didn't work. Looks like it's error on ACRE's part, can't do anything about it.

**Comment by Drofseh** [comment]
I've talked to a couple of the acre guys and it might be implemented in the future.
ACRE2 is open source on github now as well, so it might be possible to make your own function.

**Comment by Drofseh** [comment]
ACRE now has channels for the PRC-77
https://github.com/IDI-Systems/acre2/pull/392

**Comment by TrainDoctor** [comment]
> 
> 
> ACRE now has channels for the PRC-77
> [IDI-Systems/acre2#392](https://github.com/IDI-Systems/acre2/pull/392)

This issue should be marked as resolved.

**Comment by Drofseh** [comment]
I agree.
If the framework's acre_setup module works to set the -77's channels then it can be considered resolved.

**Comment by EM-Creations** [comment]
> I agree.
> If the framework's acre_setup module works to set the -77's channels then it can be considered resolved.

I don't think it works with FNC_SetRadio still annoyingly.. I might try and fix this.
